chromium_src/chrome/browser/about_flags.cc

```diff
@@ -4,6 +4,9 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "brave/components/ntp_sponsored_images/browser/features.h"
+// Include from chromium_src before about_flags.h
+#include "../../components/flags_ui/flags_state.h"
```

Comment:
I'm not sure if the code or the comment is wrong, but you use relative paths to get the chrome src, not the chromium_src

Reply:
Fixed by 09f9c4647bab0c2a546e503b0e7bfb5e59831959. The include wasn't needed at all either way. about_flags.h already includes flags_state.h.